addons/snailmail/models/snailmail_letter.py

```diff
@@ -262,6 +278,11 @@ def snailmail_print(self):
     def cancel(self):
         self.write({'state': 'canceled'})
 
+    @api.multi
+    def unlink(self):
+        self.mapped('activity_id').unlink()
+        super(SnailmailLetter, self).unlink()
```

Comment:
don't forget the return here. Even if the result is the same for this specific action. we generaly want to keep the same return API project-wide :)